renderer/Image_files.cpp

```diff
@@ -583,19 +583,20 @@ static void LoadTGA( const char *name, byte **pic, int *width, int *height, ID_T
 	targa_header.colormap_type = *buf_p++;
 	targa_header.image_type = *buf_p++;
 	
-	targa_header.colormap_index = LittleShort ( *(short *)buf_p );
-	buf_p += 2;
-	targa_header.colormap_length = LittleShort ( *(short *)buf_p );
-	buf_p += 2;
+	auto const get_little_short = [&buf_p]() {
```

Comment:
This might bump the language version, might be safer to just inline the memcpys.

Reply:
It will be for just a bit longer - I'm currently building with the CentOS 7 compiler, will be updating to Rocky 8 in April.

Reply:
CentOS 7, if I am not wrong, should have GCC 4.8, which supports C++11. Am I right?

Reply:
With an explicit flag it works, I would just need to test everything to make sure it produces what we want vs. just inlining the copies.

Reply:
I bumped the minimal version of CMake to 3.1 and I added the C++11 flag (the bump of CMake is required to avoid using custom arguments). Check if the bump is acceptable, I think it should because CMake 3.1 should be _pretty_ old nowadays to be included almost everywhere. But better safe than sorry :wink: .